scripts/moi

```diff
@@ -19,6 +21,46 @@ def _is_uuid(test):
         return True
 
 
+def _python_exec(cmd, *args, **kwargs):
+    """Wrap for exec such that a return statement can be honored"""
+    exec('def execwrapper(): %s' % cmd)
+    return execwrapper()  # noqa
+
+
+def _system_exec(cmd, *args, **kwargs):
+    """Execute a system call"""
+    from moi.job import system_call
+    return system_call(cmd, **kwargs)
+
+
+def _dump_job_detail(node, summary=False):
+    """Pretty print job detail"""
+    # format the traceback so it is pleasant looking
+    tb = 'Traceback (most recent call last):\n'
```

Comment:
You can probably move this 4 lines to the else statement, since the `node['result']` is only needed on the else branch.